client/client.go

```diff
@@ -162,6 +162,11 @@ type Client interface {
 	// this may differ from the initial Endpoints provided in the Config.
 	Endpoints() []string
 
+	// RestEndpoints resets the set of API endpoints used by Client to resolve
```

Comment:
I feel Reset is slightly better as this is a client Interface and we want the func name to be more specific.